CMakeLists.txt

```diff
@@ -856,6 +856,10 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
 		DESTINATION ${CMAKE_INSTALL_PREFIX}/share/pixmaps)
 	install(FILES Installer/dolphin-emu.desktop
 		DESTINATION ${CMAKE_INSTALL_PREFIX}/share/applications)
+	# Install udev rule
+	install(FILES Installer/51-gcadapter.rules
+		DESTINATION /lib/udev/rules.d
```

Comment:
The correct place (where udev reads them from) is /usr/lib/udev/rules.d

Most distros now link from /usr/lib to /lib, but incase there are still some which don't, /usr/lib is the correct place.